### ISSUE TITLE
Support for older browsers

### ISSUE THREAD

**Issue by RedSparr0w** [imyelo/pokequest-wiki#1]
Possibly add support for older browsers?

Errors on iOS 9.3.3:
```console
SyntaxError: Unexpected keyword 'const'. Const declarations are not supported in strict mode.
0.291345cf.js:1

{
    "line": 1,
    "column": 1997,
    "sourceURL": "https://www.pokequest.wiki/static/js/app.e7ff5d2e.js",
    "type": "missing",
    "request": "https://www.pokequest.wiki/static/js/0.291345cf.js"
}
app.e7ff5d2e.js:1

"Unhandled promise rejection" {
    "line": 1,
    "column": 1997,
    "sourceURL": "https://www.pokequest.wiki/static/js/app.e7ff5d2e.js",
    "type": "missing",
    "request": "https://www.pokequest.wiki/static/js/0.291345cf.js"
}
chunk-vendors.8824f392.js:16
```

**Comment by imyelo** [comment]
It seems that the release of [ml-combinations](https://npm.im/ml-combinations) did not compiled to es5. Let me try to handle it.

**Comment by RedSparr0w** [comment]
Thanks, much appreciated  😁

**Comment by imyelo** [comment]
Should have been fixed in the latest version (20180726.1).

Feel free to reopen it and let me know if anything wrong. 😉

**Comment by RedSparr0w** [comment]
Working perfectly!
Thanks for the update 👍
